deltachat-rpc-client/tests/test_something.py

```diff
@@ -801,3 +801,29 @@ def test_get_all_accounts_deadlock(rpc):
         all_accounts = rpc.get_all_accounts.future()
         rpc.add_account()
         all_accounts()
+
+
+def test_delete_deltachat_folder(acfactory, direct_imap):
+    """Test that DeltaChat folder is recreated if user deletes it manually."""
+    addr, password = acfactory.get_credentials()
+    ac1 = acfactory.get_unconfigured_account()
+    ac1.add_transport({"addr": addr, "password": password})
+    ac1.set_config("mvbox_move", "1")
+    ac1.bring_online()
```

Comment:
Why not use `new_configured_account()` here? This is calling `set_config()` after `add_transport()`, so the first 3 lines could just be replaced by `new_configured_account()`. And `new_configured_account()` doesn't call `bring_online()`.